frontend/src/routers/AppRouter/AppRouter.tsx

```diff
@@ -221,24 +221,17 @@ export const AppRouter = () => {
 				<WithSessionSearchContext>
 					<WithErrorSearchContext>
 						{(isNewWorkspacePage || isNewProjectPage) &&
-							(isLoggedIn ? (
-								<NewProjectPage
-									workspace_id={
-										isNewProjectPage
-											? workspaceId
-											: undefined
-									}
-								/>
-							) : (
-								<Navigate to={SIGN_IN_ROUTE} />
-							))}
+						!isLoggedIn ? (
+							<Navigate to={SIGN_IN_ROUTE} />
+						) : null}
 						<Routes>
+							<Route path="/new" element={<NewProjectPage />} />
 							<Route
-								path="/new"
+								path="/w/:workspace_id/new"
 								element={
-									<Landing>
-										<SwitchWorkspace />
-									</Landing>
+									<NewProjectPage
+										workspace_id={workspaceId}
+									/>
```

Comment:
Again these changes are to have the background app as is, while showing the component itelf as the new modal.

Reply:
the problem with your implementation is that the new modal is overlayed on the existing `/new` modal, rather than the page the user clicks to `/new` from
![image](https://github.com/highlight/highlight/assets/1351531/546713c4-cd9a-4f90-8829-2e08bb647800)

Reply:
fixed to be an overlay in 06092b230bfe2adace80ee86d435835ec492e96f